### PR TITLE
event is 90% complete in getting reduxified

### PR DESCRIPTION
**Pull request by jcee-io**
it is for the most part, everything is operational EXCEPT that it doesnt save to the database (this is due to the lack of call after the upvote) will fix tomorrow morning since i was too sleepy to implement it tonight

### CODE REVIEW

File: client/src/Events.jsx [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-    } else {
-      this.setState({
-        votes: this.state.votes -= 1,
-      }, this.patchVotesInDB);
-    }
+  componentDidMount() {
+    const { event, setVote } = this.props;
+    setVote(event.votes || 0, event._id);
+    this.votes = this.props.vote;
+    console.log(this.props.vote);
```
Comment (chalkera) [inline client/src/Events.jsx:28]: Remove later

File: server/index.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ app.post('/entry', ({ body }, response) => {
 });
 
 app.put('/entry', ({ body }, response) => {
+  console.log(body);
```
Comment (chalkera) [inline server/index.js]: Definitely need to remove this

**Review by chalkera — APPROVED** [review]
Just remember to remove any console logs later on.

**Review by chalkera — APPROVED** [review]

**Review by kfang1220 — COMMENTED** [review]
Looks good, i can't comment much on redux changes.

**Pull request merged**